src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php

```diff
@@ -76,4 +77,16 @@ function (Mage_Sales_Model_Quote_Address_Total $total) {
             )
         );
     }
+    
+    /**
+     * @param Mage_Sales_Model_Quote $quote
+     * @return string
+     */
+    public function addCartPriceDisplaySettings(Mage_Sales_Model_Quote $quote)
+    {
+        $quote->setData(
+            self::KEY_CART_PRICE_DISPLAY_SETTINGS,
+            Mage::getStoreConfig(Mage_Tax_Model_Config::XML_PATH_DISPLAY_CART_PRICE)
```

Comment:
You will need to set a store for this setting, else magento will default to what is currently the global store. You don't want that inside of endpoint requests as they default to "admin" store.

Currently it works because the endpoint that calls this sets the store globally. Since it's a helper, others endpoints may not have this setup like that.